<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <title>Quantitative Genetics</title>
  <meta name="description" content="">
  <meta name="author" content="Antonio Augusto F Garcia">
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
  <link rel="stylesheet" href="http://netdna.bootstrapcdn.com/bootswatch/2.3.1/amelia/bootstrap.min.css">
<link rel="stylesheet" href="./frameworks/revealjs/css/reveal.min.css">
  <link rel="stylesheet" href="./frameworks/revealjs/css/theme/night.css" id="theme">
  <link rel="stylesheet" href="./highlighters/highlight.js/css/zenburn.css" id="theme">
  <!--[if lt IE 9]>
  <script src="lib/js/html5shiv.js"></script>
  <![endif]-->  <link rel="stylesheet" href = "assets/css/slidify-reveal.css">

</head>
<body>
  
  <div class="reveal">
    <div class="slides">
      <section class='' data-state='' id='slide-1'>
  <h1>Quantitative Genetics</h1>
  <h2>Resemblance Between Relatives</h2>

<p><small>
Instructor: <a href="http://about.me/augusto.garcia">A Augusto F Garcia</a>
/ <a href="http://statgen.esalq.usp.br">Statistical Genetics Lab</a> </small></p>

<p><small>Department of Genetics, Luiz de Queiroz College of Agriculture,
University of SÃ£o Paulo (Brazil)</small></p>

<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>

  <aside class='notes'>
    <p>Some notes on the first slide</p>

  </aside>
</section>
<section class='' data-state='' id='slide-2'>
  <h1>Content</h1>
  <h3>Introduction</h3>

<h3>Measures of Relatedness</h3>

<h3>The Genetic Covariance Between Relatives</h3>

<h3>The Heritability Concept</h3>

</section>
<section class='' data-state='' id='slide-3'>
  <h1>Introduction</h1>
  <ul>
<li>Assuming no \(G \times E\)</li>
<li>\(x\) and \(y\): members of a particular relationship (such as
half-sibs, full-sibs, etc)</li>
<li>Phenotypes:
\[z_x=G_x+E_x+e_x\]
\[z_y=G_y+E_y+e_y\]</li>
<li>The phenotypic covariance between relatives \(x\) and \(y\) is
\[\sigma_z(x,y)=\sigma[(G_x+E_x+e_x),(G_y+E_y+e_y)]\]
\[\phantom{AAAAAAAAAa}=\sigma_G(x,y)+\sigma_{G,E}(x,y)+\sigma_{G,E}(y,x)+\sigma_E(x,y)\]</li>
<li>Assuming no genotype-environment covariance
\[\sigma_z(x,y)=\sigma_G(x,y)+\sigma_E(x,y)\]</li>
</ul>

<blockquote>
<p>\(\sigma_G(x,y)\) is a natural consequence of relatives inheriting
copies of the same genes</p>
</blockquote>

</section>
<section class='' data-state='' id='slide-4'>
  <h1>Measures of Relatedness</h1>
  <ul>
<li>Relatedness is defined in terms of an specific frame of reference</li>
<li><strong><em>Identity by descent</em></strong>: genes <em>ibd</em> are direct descendants of a
specific gene carried by some ancestor</li>
<li><strong><em>Identity in state</em></strong>: identical genes, but not coming from an
common ancestor</li>
<li><strong><em>Inbred</em></strong>: individuals that contain pairs of alleles that are
<em>ibd</em></li>
<li>There are 15 possible configurations of identity by descent</li>
</ul>

</section>
<section class='' data-state='' id='slide-5'>
  
  <p><img src="./images/ibd_ibs.png" alt=""></p>

</section>
<section class='' data-state='' id='slide-6'>
  
  <p><img src="./images/All_ibd.png" alt=""></p>

</section>
<section class='' data-state='' id='slide-7'>
  
  <ul>
<li>The values for \(\Delta\) depends on the relationship between \(x\) and
\(y\)</li>
<li>For example, if \(x\) is a non-inbred parent (mother or father) and
\(y\) is its non-inbred progeny (parent-offspring relationship),
\(\Delta_8=1\) and all other \(\Delta_i=0\)</li>
</ul>

<p><img src="./images/delta8.png" alt=""></p>

</section>
<section class='' data-state='' id='slide-8'>
  
  <ul>
<li>For \(x\) and \(y\) as non-inbred full-sibs</li>
<li>\(0.5\) probability of receiving the same paternal gene</li>
<li>\(0.5\) probability of receiving the same maternal gene</li>
<li>\(\Delta_7=0.25\)</li>
<li>\(\Delta_8=0.50\)</li>
<li>\(\Delta_9=0.25\)</li>
<li>All other \(\Delta_i\)&#39;s are \(0\)</li>
</ul>

<p><img src="./images/delta7_8_9.png" alt=""></p>

</section>
<section class='' data-state='' id='slide-9'>
  <h2>Coefficient of Coancestry</h2>
  <ul>
<li>Suppose that single genes are drawn randomly from individuals \(x\)
and \(y\)</li>
<li>The probability that these genes are <em>ibd</em> is the <strong><em>coefficient of
coancestry</em></strong> \(\Theta_{xy}\)</li>
</ul>

<p>\[\Theta_{xy}=\Delta_1+\frac{1}{2}(\Delta_3+\Delta_5+\Delta_7)+\frac{1}{4}\Delta_8\]</p>

</section>
<section class='' data-state='' id='slide-10'>
  <h2>Coefficient of Inbreeding</h2>
  <ul>
<li>Assume now that we have an offspring \(z\) from \(x\) and \(y\)</li>
<li>\(\Theta_{xy}\) is the probability that the two genes at a locus in
\(z\) are identical by descent</li>
<li>This is Wright&#39;s (1922) <strong><em>inbreeding coefficient</em></strong> \(f_z\)</li>
<li>Notice that \(f_z\), next generation, is equivalent to \(\Theta_{xy}\),
previous generation
\[f_z=\Theta_{xy}\]</li>
</ul>

</section>
<section class='' data-state='' id='slide-11'>
  <h3>Some useful derivations</h3>
  <ul>
<li>Individual with itself
\[\Theta_{xx}=\frac{1}{2}(1+f_x)\]</li>
<li>Parent-offspring
\[\Theta_{po}=\frac{1}{4}(1+f_p+2f_o)\]</li>
<li>Full-sibs in general (\(m\): mother, \(f\): father)
\[\Theta_{xy}=\frac{1}{8}(2+f_m+f_f+4\Theta_{mf})\]</li>
<li>Full-sibs, random mating
\[\Theta_{xy}=\frac{1}{4}\]</li>
</ul>

</section>
<section class='' data-state='' id='slide-12'>
  <h2>Coefficient of Fraternity</h2>
  <ul>
<li>The probability that single-locus genotypes (both genes) of two
individuals are <em>ibd</em></li>
<li>Notation: mothers of \(x\) and \(y\) are \(m_x\) and \(m_y\), fathers are
\(f_x\) and \(f_y\)</li>
<li>Therefore:
\[\Delta_{xy}=\Theta_{m_xm_y}\Theta_{f_xf_y}+\Theta_{m_xf_y}\Theta_{f_xm_y}\]</li>
<li>This is the same of
\[\Delta_{xy}=\Delta_1+\Delta_7\]</li>
<li>(No inbreeding, \(\Delta_{xy}=\Delta_{7}\))</li>
</ul>

<p><img src="./images/delta1.png" alt=""></p>

<p><img src="./images/delta7.png" alt=""></p>

</section>
<section class='' data-state='' id='slide-13'>
  
  <p><img src="./images/Table7_1.png" alt=""></p>

</section>
<section class='' data-state='' id='slide-14'>
  <h1>The Genetic Covariance Between Relatives</h1>
  <ul>
<li>First, let us make a number of assumptions, including

<ul>
<li>Diploid, autosomal loci</li>
<li>Random mating</li>
<li>No linkage disequilibrium</li>
<li>No \(G \times E\) and no covariance \(G,E\)</li>
</ul></li>
<li>Then, let us take two individuals in a
population where all members have the same relationship</li>
</ul>

</section>
<section class='' data-state='' id='slide-15'>
  
  <p>\[G_{ijkl..}(x)=\mu_G+[\alpha^x_i+\alpha^x_j+\alpha^x_k+\alpha^x_l+...]+[\delta^x_{ij}+\delta^x_{kl}+...]\]</p>

<p>\[\phantom{AAAAAA}+[(\alpha\alpha)^x_{ik}+(\alpha\alpha)^x_{il}+(\alpha\alpha)^x_{jk}+(\alpha\alpha)^x_{jl}+...]\]</p>

<p>\[\phantom{AAAAAAAAAA}+[(\alpha\delta)^x_{ikl}+(\alpha\delta)^x_{jkl}+(\alpha\delta)^x_{kij}+(\alpha\delta)^x_{lij}+...]+(\delta\delta)^x_{ijkl}+...\]</p>

<p>\[\phantom{ZZZ}\]</p>

<p>\[G_{ijkl..}(y)=\mu_G+[\alpha^y_i+\alpha^y_j+\alpha^y_k+\alpha^y_l+...]+[\delta^y_{ij}+\delta^y_{kl}+...]\]</p>

<p>\[\phantom{AAAAAA}+[(\alpha\alpha)^y_{ik}+(\alpha\alpha)^y_{il}+(\alpha\alpha)^y_{jk}+(\alpha\alpha)^y_{jl}+...]\]</p>

<p>\[\phantom{AAAAAAAAAA}+[(\alpha\delta)^y_{ikl}+(\alpha\delta)^y_{jkl}+(\alpha\delta)^y_{kij}+(\alpha\delta)^y_{lij}+...]+(\delta\delta)^y_{ijkl}+...\]</p>

</section>
<section class='' data-state='' id='slide-16'>
  
  <ul>
<li>Fisher (1918) showed that
\[\sigma_G(x,y)=\sigma_A(x,y)+\sigma_D(x,y)+\sigma_{AA}(x,y)+\sigma_{AD}(x,y)+\sigma_{DD}(x,y)+...\]</li>
<li>This is so because the genetic effects are uncorrelated within
individuals (as discussed before) and also between individuals</li>
</ul>

</section>
<section class='' data-state='' id='slide-17'>
  
  <ul>
<li>The covariance between \(x\) and \(y\) caused by <strong>additive effects</strong> is
\[E[(\alpha^x_i+\alpha^x_j)(\alpha^y_i+\alpha^y_j)]\]

<ul>
<li>\(E(\alpha^x_i\alpha^y_i)\) is one of the terms</li>
<li>The only way this expectation is not zero is if allele \(i\) is
the same in individuals \(x\) and \(y\)</li>
<li>If this happens, \(E(\alpha^x_i\alpha^y_i)=E(\alpha^2_i)\) (half
the additive genetic variance)</li>
<li>But the probability of \(i\) being the same is \(\Theta_{xy}\)</li>
<li>With the same reasoning for the four terms, we have that the
expectation is 
\[4\Theta_{xy}E(\alpha^2_i)\] 
(twice the additive genetic variance for this loci times \(\Theta_{xy}\))</li>
</ul></li>
</ul>

</section>
<section class='' data-state='blue' id='slide-18'>
  <h3>Additive Genetic Covariance</h3>
  <ul>
<li>By summing over loci</li>
</ul>

<p>\(\phantom{ZZ}\)</p>

<p>\[\sigma_A(x,y)=2\Theta_{xy}\sigma^2_A\]</p>

</section>
<section class='' data-state='' id='slide-19'>
  <h3>Dominance Genetic Covariance</h3>
  <ul>
<li>\(E(\delta^x_{ij}\delta^y_{ij})\)</li>
<li>If \(x\) and \(y\) are <em>ibd</em> for both genes at this locus, then
\[E(\delta^x_{ij}\delta^y_{ij})=E(\delta^2_{ij})\]</li>
<li>This is the dominance genetic variance</li>
<li>This identity has probability \(\Delta_{xy}\)</li>
<li>For all loci
\[\sigma_D(x,y)=\Delta_{xy}\sigma^2_D\]</li>
</ul>

</section>
<section class='' data-state='' id='slide-20'>
  <h3>Epistatic Genetic Covariance</h3>
  <ul>
<li>Using the same reasoning, it is possible to show that
\[\sigma_{AA}(x,y)=(2\Theta_{xy})^2\sigma^2_{AA}\]</li>
</ul>

</section>
<section class='' data-state='blue' id='slide-21'>
  <h2>Covariance Between Relatives</h2>
  <p>\[\phantom{ZZ}\]</p>

<p>\[\sigma_G(x,y)=2\Theta_{xy}\sigma^2_A+\Delta_{xy}\sigma^2_D+(2\Theta_{xy})^2\sigma^2_{AA}\]
\[\phantom{ZZ}\]
\[\phantom{AAAAAAAAAAAA}+2\Theta_{xy}\Delta_{xy}\sigma^2_{AD}+\Delta^2_{xy}\sigma^2_{DD}+(2\Theta_{xy})^3\sigma^2_{AAA}+...\]</p>

</section>
<section class='' data-state='' id='slide-22'>
  
  <p><img src="./images/Table7_2.png" alt=""></p>

<ul>
<li>This can be used to estimate variance components based on
covariances</li>
</ul>

</section>
<section class='' data-state='' id='slide-23'>
  <h1>The Heritability Concept</h1>
  <ul>
<li>The ratio \(\frac{\sigma^2_A}{\sigma^2_z}\) is known as the
<strong>narrow-sense heritability</strong></li>
<li>This is an attempt to have a parameter that describes the genetic
resemblance between parents and offspring</li>
<li>But notice that the expression for genetic covariance between
relatives has the term \(2\Theta_{xy}\sigma^2_A\)</li>
<li>Thus, under the assumption that the additive genetic variance is the
most important source of phenotypic covariance:
\[h^2\simeq\frac{Cov(z_x,z_y)}{2\Theta_{xy}Var(z)}\]</li>
<li>This is a good approximation to the heritability</li>
</ul>

</section>
<section class='' data-state='indigo' id='slide-24'>
  <h1>Homework</h1>
  <ul>
<li>Read Chapter 7 of Lynch and Walsh Book</li>
</ul>

</section>
    </div>
  </div>
</body>
  <script src="./frameworks/revealjs/lib/js/head.min.js"></script>
  <script src="./frameworks/revealjs/js/reveal.min.js"></script>
  <script>
  // Full list of configuration options available here:
  // https://github.com/hakimel/reveal.js#configuration
  Reveal.initialize({
    controls: true,
    progress: true,
    history: true,
    center: true,
    theme: Reveal.getQueryHash().theme || 'night', 
    transition: Reveal.getQueryHash().transition || 'linear', 
    dependencies: [
    // Cross-browser shim that fully implements classList -
    // https://github.com/eligrey/classList.js/
      { src: './frameworks/revealjs/lib/js/classList.js', condition: function() { return !document.body.classList;}},
      // Zoom in and out with Alt+click
      { src: './frameworks/revealjs/plugin/zoom-js/zoom.js', async: true, condition: function() { return !!document.body.classList; } },
      // Speaker notes
      { src: './frameworks/revealjs/plugin/notes/notes.js', async: true, condition: function() { return !!document.body.classList; } },
      // Remote control your reveal.js presentation using a touch device
      //{ src: './frameworks/revealjs/plugin/remotes/remotes.js', async: true, condition: function() { return !!document.body.classList; } }
      ]
  });
  </script>  <!-- MathJax: Fall back to local if CDN offline but local image fonts are not supported (saves >100MB) -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [['$','$'], ['\\(','\\)']],
      processEscapes: true
    }
  });
</script>
<script type="text/javascript" src="http://cdn.mathjax.org/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
<!-- <script src="https://c328740.ssl.cf1.rackcdn.com/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script> -->
<script>window.MathJax || document.write('<script type="text/x-mathjax-config">MathJax.Hub.Config({"HTML-CSS":{imageFont:null}});<\/script><script src="./widgets/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><\/script>')
</script>
<!-- LOAD HIGHLIGHTER JS FILES -->
<script src="./highlighters/highlight.js/highlight.pack.js"></script>
<script>hljs.initHighlightingOnLoad();</script>
<!-- DONE LOADING HIGHLIGHTER JS FILES -->
 

</html>